src/Hosting/Hosting/test/WebHostBuilderTests.cs

```diff
@@ -139,6 +139,30 @@ public async Task StartupConfigureThrows_Fallback(IWebHostBuilder builder)
             }
         }
 
+        [Theory]
+        [MemberData(nameof(DefaultWebHostBuilders))]
+        public async Task StartupConfigureThrowsDoesNotThrowTargetInvocationException(IWebHostBuilder builder)
+        {
+            var server = new TestServer();
+            var host = builder.UseServer(server).UseStartup<StartupConfigureThrows>().Build();
+            using (host)
+            {
+                var exception = await Assert.ThrowsAsync<Exception>(() => host.StartAsync());
+                Assert.Equal("Exception from Configure", exception.Message);
+            }
+        }
+
+        [Theory]
+        [MemberData(nameof(DefaultWebHostBuilders))]
+        public void StartupConfigureServicesThrowsDoesNotThrowTargetInvocationException(IWebHostBuilder builder)
+        {
+            var server = new TestServer();
+            var hostBuilder = builder.UseServer(server).UseStartup<StartupConfigureServicesThrows>();
```

Comment:
This hits `void` code path, add another one that hits `IServiceProvider` one.

Reply:
The other tests (StartupManagerTests) hit all code paths (because it was exploding before I got it right).